site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -207,6 +207,18 @@ go run cmd/sdk-server/main.go --local
 
 Commandline flags (e.g. `--local`) are exactly the same as command line flags when utilising a pre-built binary.
 
+**Enabling Specific Features:**
+For development and testing purposes, you might want to enable specific features in the SDK Server. To do this, set the FEATURE_GATES environment variable before running the server. For instance:
+```bash
+FEATURE_GATES="FeatureName=true" go run cmd/sdk-server/main.go --local
+```
+
+Replace `FeatureName` with the name of the feature you want to enable. This command starts the local SDK Server with the specified feature activated.
```

Comment:
Let's create move this right after the section "Running In “Local Mode".

````suggestion

## Enabling Feature Gates

For development and testing purposes, you might want to enable specific [features gates]({{< relref "../feature-stages.md#feature-gates" >}}) in the local SDK Server. 

To do this, you can either set the `FEATURE_GATES` environment variable or use the `--feature-gates` command line parameter like so, with the same format as utilised when [configuring it on a Helm install]({{< ref "/docs/Installation/Install Agones/helm.md#configuration" >}}).

For example:

```bash
./sdk-server.linux.amd64 --local --feature-gates Example=true
```
or 
```bash
FEATURE_GATES=Example=true ./sdk-server.linux.amd64 --local
```


````

You'll probably have to fix my link paths though - I did this off the top of my head.

Reply:
Moved `Enabling Feature Gates` section to immediately follow the `Running In “Local Mode"`